Dockerfile

```
@@ -0,0 +1,58 @@
+ARG BASE_TAG=17-jdk-alpine
+
+FROM eclipse-temurin:$BASE_TAG
+LABEL maintainer="Jahia"
+
+ARG REFRESHED_AT
+ENV REFRESHED_AT=$REFRESHED_AT
+
+ARG MAVEN_VERSION=3.9.11
+ENV MAVEN_HOME=/opt/maven \
+  MAVEN_CONFIG=/root/.m2
+
+RUN echo @new-stable https://dl-cdn.alpinelinux.org/alpine/v3.21/community >> /etc/apk/repositories \
+  && echo @new-stable https://dl-cdn.alpinelinux.org/alpine/v3.21/main >> /etc/apk/repositories \
```

Out of curiosity, where does this "3.21" version come from?
Would that make sense to use 
```suggestion
RUN echo @new-stable https://dl-cdn.alpinelinux.org/alpine/latest-stable/community >> /etc/apk/repositories \
  && echo @new-stable https://dl-cdn.alpinelinux.org/alpine/latest-stable/main >> /etc/apk/repositories \
```
instead?

---

Yes we could do that, the 3.21 was coming from the repos I used for inspiration: https://github.com/timbru31/docker-java-node/blob/master/17/jdk/22/alpine/Dockerfile

But if all of the rest is on latest, it's probably good to do that as well.